src/components/ui/02-molecules/bottom-banner.jsx

```diff
@@ -0,0 +1,310 @@
+    position: "relative",
+    pl: 0,
+    pt: 0,
+    maxWidth: ["100%", '100%', "40% !important"],
+    
+    '& div': {
+      width: ["255px", "322.67px", "100%",],
+      height: ["155px", "235.52px", "100%"],
+      m: '0 auto',
+    },
+
+    "& svg": {
+      display: "block",
+      position: "relative",
+      width: "auto",
+      height: "auto",
+    },
+  };
+
+  return (
+    <>
+      <ButtonBase sx={bottomButtonStyles} onClick={handleClickOpen}>
+        <Typography variant="large-text" gutterBottom sx={{ mb: 0 }}>
+          Quilted by Four Kitchens
+        </Typography>
+      </ButtonBase>
+      <Dialog
+        open={open}
+        TransitionComponent={Transition}
+        scroll={scroll}
+        onClose={handleClose}
+        aria-describedby="bottom-banner-dialog"
+        sx={mainContainerStyles}
+      >
+        <DialogContent sx={dialogContentStyles}>
+          <Stack sx={patternWrapperStyles}>
+            <BackgroundPattern />
+          </Stack>
+          <Box sx={containerStyles}>
+            <Grid container spacing={3} sx={gridStyles}>
+              <Grid
+                item
+                xs={12}
+                md={6}
+                sx={{ ...gridImageStyles, gridItemStyles }}
+              >
+                <Logo>
+                  <Team />
+                </Logo>
+              </Grid>
+              <Grid item xs={12} md={6} sx={gridItemStyles}>
+                <Typography
+                  variant="h2"
+                  gutterBottom
+                  sx={{ mb: 1 }}
+                  id="bottom-banner-dialog"
+                >
+                  Keeping pace with technology is hard work
+                </Typography>
+                <Typography variant="default-text" gutterBottom>
+                  Our experience in education, advocacy, and publishing informs
+                  our unique, content-focused approach to crafting websites that
+                  make an impact and stand the test of time. Whether you’re
+                  looking for extra hands or a complete redesign, we can help
+                  your team realize its potential.
+                </Typography>
+                <Stack spacing={2} direction="row" sx={{ mt: 2, zIndex: 2, position: 'relative' }}>
+                  <Button
+                    variant="contained"
+                    color="electric-orange"
+                    href="https://www.fourkitchens.com/contact/"
+                    sx={buttonStyles}
+                  >
+                    Let’s Talk
+                  </Button>
+                </Stack>
+              </Grid>
+            </Grid>
+            <Box sx={creditsContainerStyles}>
+              <Typography variant="small-text" sx={creditsTextStyles}>
+                Credits: Adam Erickson, Ashley Hitson, Hector Lopez, Joanna Cendrowski, Katy Fernández, 
+                Kelvin Cheng, Laura Johnson, Mari Núñez, Mike Goulding, Randy Dean West, Roberto Hernández, 
```

Comment:
Please change "West" to "Oest". Thanks! (I'm used to this happening with autocorrect.)

Reply:
Omg I'm really sorry! Fixed